rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -101,20 +181,473 @@ def test_hop_optimism_eth_receive_no_event(database, optimism_inquirer, optimism
         database=database,
         tx_hash=tx_hash,
     )
+    bridge_amount = '0.03958480553397407'
+    timestamp = TimestampMS(1666977475000)
     user_address = optimism_accounts[0]
     expected_events = [
         EvmEvent(
             tx_hash=tx_hash,
             sequence_index=0,
-            timestamp=TimestampMS(1666977475000),
+            timestamp=timestamp,
             location=Location.OPTIMISM,
             event_type=HistoryEventType.WITHDRAWAL,
             event_subtype=HistoryEventSubType.BRIDGE,
             asset=A_ETH,
-            balance=Balance(amount=FVal('0.03958480553397407')),
+            balance=Balance(amount=FVal(bridge_amount)),
             location_label=user_address,
-            notes='Bridge 0.03958480553397407 ETH via hop protocol',
+            notes=f'Bridge {bridge_amount} ETH via Hop protocol',
             counterparty=CPT_HOP,
-            address=string_to_evm_address('0x86cA30bEF97fB651b8d866D45503684b90cb3312'),
-        )]
+            address=ETH_AMM_WRAPPER_OPTIMISM,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xc009D690942DbAaC8d5b15B20EFb24fCbFF77Ddd']])
+def test_hop_usdc_bridge(database, ethereum_inquirer, ethereum_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xdca89892d6a738c2ea278d93a5e1757b2f946f4e24c4aa02d59a0abf6a8e5f4b')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1715038763000)
+    bridge_amount = '9006.683634'
+    user_address = ethereum_accounts[0]
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=167,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_USDC,
+            balance=Balance(amount=FVal(bridge_amount)),
+            location_label=user_address,
+            notes=f'Bridge {bridge_amount} USDC via Hop protocol',
+            counterparty=CPT_HOP,
+            address=USDC_BRIDGE,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('optimism_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+def test_hop_eth_bridge_optimism(database, optimism_inquirer, optimism_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x4f1e95506c10f061ddfe28a7437f3b651959ff17f1e2a7a148c8896147ee357e')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=optimism_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1643122781000)
```

Comment:
all in one line. Okay will stop repeating this. Just like with the other comments I added above, repeat it for all tests you wrote here